.gitattributes

```diff
@@ -1,6 +1,13 @@
+# Default behaviour, for if core.autocrlf isn't set
+* text=auto
+
+# Binary files
 boot/ocamlc binary
 boot/ocamllex binary
 boot/ocamldep binary
+*.gif
+*.png
+*.tfm
```

Comment:
`manual/manual/library/libgraph.png`, `manual/styles/*.tfm`, `manual/htmlman/*.gif`. I figured that they're universal enough extensions (certainly `.png` and `.gif`, anyhow) to add them generically but each wildcard version could be replaced with that more precise location if preferred?